controllers/platform/platformutils.go

```diff
@@ -38,6 +38,11 @@ import (
 type ResourceCustomizer func(object ctrl.Object) ctrl.Object
 
 func ConfigureRegistry(ctx context.Context, c client.Client, p *operatorapi.KogitoServerlessPlatform, verbose bool) error {
+	if p.Spec.BuildPlatform.BuildStrategy == operatorapi.PlatformBuildStrategy && p.Status.Cluster == operatorapi.PlatformClusterOpenShift {
+		p.Spec.BuildPlatform.Registry = operatorapi.RegistrySpec{}
+		return nil
+	}
```

Comment:
When implementing https://issues.redhat.com/browse/KOGITO-9218, we should notify the user in the status that the Registry is being ignored on OpenShift using the platform build strategy.

Reply:
I'll add a @TODO